design-documents/graph-ql/coverage/PaginationChangesForEntities.graphqls

```diff
@@ -0,0 +1,25 @@
+type Cart {
```

Comment:
When possible, please locate the original proposal with the schema and modify it there (like it is done for `customer-orders`). Thanks